internal/cmd/controller/reconciler/bundle_controller.go

```diff
@@ -1008,3 +992,50 @@ func upper(op controllerutil.OperationResult) string {
 		return "Unknown"
 	}
 }
+
+// BundleDeploymentMapFunc returns a function that maps a BundleDeployment to a Bundle.
+func BundleDeploymentMapFunc(r *BundleReconciler) func(ctx context.Context, a client.Object) []reconcile.Request {
```

Comment:
nit: we don't need to export this function.

Reply:
I did it this way because the `bundle_controller_test.go` is in the `reconciler_test` package.
I can also add a specific file just to test this part of the code... I thought that was adding too many changes for an export in an internal package.